### DISCUSSION TITLE
Filter - table

### DISCUSSION THREAD

**Discussion by EduZeroum** [Budibase/budibase#1930] [Help] [answered]
Hello i'm new at budibase and i have a question... I would like to put a filter on my table that i can see my records in a certain period of time. Explaining more in depth, the table in question will be to list customers who have a license that has time, and the objective was to select for example June 1st to June 21st and that will show users with that period of time in table. I tried date range but it only gives me "default" date limits

Sorry my bad english, i will put a print of my table , might help.
![Table](https://user-images.githubusercontent.com/87012689/124583044-87124600-de4a-11eb-9375-114e00331999.png)

**Comment by joebudi** [comment]
Hey @EduZeroum 

You cannot create a filter within the table, but you can create a filter using the data provider component.

[This video ](https://www.loom.com/share/2ddc5943593d428c8405b3c52a80b8d9
)demonstrates how.

In your case, you would have to use 2 date pickers, and within your filter add two expressions; more than Date picker A. and less than date picker B.

I would like to flag, that filter with a date picker is currently not working, but @aptkingston fixed the [issue](https://github.com/Budibase/budibase/issues/1931), and will release it asap.

**Comment by aptkingston** [comment, marked as answer]
Here's a quick screenshot showing how your components should be layed out to achieve this:
![image](https://user-images.githubusercontent.com/9075550/124601845-3574b600-de60-11eb-9238-341a8bc5dacd.png)

The only important thing to note is that the data provider must be inside the form. Everything else is flexible. As @joebudi said, this won't be possible until the fix for that issue is released.

**Comment by EduZeroum** [comment]
Thank you so much for the reply, i really apreciate it and now i can understand a bit more how budibase filters work.

I will wait till the issue is fixed.

Best regards,
Eduardo.
